quic/s2n-quic-core/src/varint/tests.rs

```diff
@@ -68,3 +68,87 @@ sequence_test!(four_byte_sequence_test(
 sequence_test!(two_byte_sequence_test([0x7b, 0xbd], 15293));
 
 sequence_test!(one_byte_sequence_test([0x25], 37));
+
+mod tests {
+    use super::*;
+    use s2n_codec::assert_codec_round_trip_value;
+
+    #[cfg(kani)]
+    #[kani::proof]
+    fn kani_one_byte_sequence_test() {
+        // https://www.rfc-editor.org/rfc/rfc9000#section-16
+        // One byte sequences have the first two MSBs encoded as 00; the
+        // last six bits are the usable values.
```

Comment:
A slight change and this will be picked up in the duvet report.

```suggestion
        //= https://www.rfc-editor.org/rfc/rfc9000#section-16
        //= type=test
        //# One byte sequences have the first two MSBs encoded as 00; the
        //# last six bits are the usable values.
```